hw/top_earlgrey/data/top_earlgrey_memory.h.tpl

```diff
@@ -0,0 +1,34 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _TOP_${top["name"].upper()}_MEMORY_H_
+#define _TOP_${top["name"].upper()}_MEMORY_H_
+
+/**
+ * @file
+ * @brief Assembler-only Generated Definitions
+ */
+
+// Include guard for assembler
+#ifdef __ASSEMBLER__
+
+/**
```

Comment:
Would you mind adding a comment that describes what *actually* gets generated? Or is the checked-in file supposed to do that instead?

Reply:
CI fails unless the checked-in file matches what is generated, and I do feel that `/* Memory base address` is a good clue, as is `/* Memory size`. I've changed some comments here though to give more exposition about the file itself though